akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowSpec.scala

```diff
@@ -40,48 +40,6 @@ class FlowSpec extends StreamSpec(ConfigFactory.parseString("akka.actor.debug.re
   val identity: Flow[Any, Any, NotUsed] ⇒ Flow[Any, Any, NotUsed] = in ⇒ in.map(e ⇒ e)
   val identity2: Flow[Any, Any, NotUsed] ⇒ Flow[Any, Any, NotUsed] = in ⇒ identity(in)
 
-  // TODO: Reenable these tests
-  //  class BrokenActorInterpreter(_shell: GraphInterpreterShell, brokenMessage: Any)
-  //    extends ActorGraphInterpreter(_shell) {
-  //
-  //    override protected[akka] def aroundReceive(receive: Receive, msg: Any) = {
-  //      msg match {
-  //        case ActorGraphInterpreter.OnNext(_, 0, m) if m == brokenMessage ⇒
-  //          throw new NullPointerException(s"I'm so broken [$m]")
-  //        case _ ⇒ super.aroundReceive(receive, msg)
-  //      }
-  //    }
-  //  }
-
-  //  val faultyFlow: Flow[Any, Any, NotUsed] ⇒ Flow[Any, Any, NotUsed] = in ⇒ in.via({
-  //    val stage = fusing.Map({ x: Any ⇒ x })
```

Comment:
getting now a Flow to materialize with a broken ActorGraphInterpreter does not work as simply as this. A new materializer phase must be created and properly implemented, emulating a faulty flow. Plus, a Flow wrapping a suitable traversal should be manually created. I am not sure all this hassle is worth it, I can't imagine anybody maintaining this mess after it is written. I am open for any clever ideas, but for now I vote for removing this and the corresponding test altogether.